apps/src/tutorialExplorer/search.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import FilterGroupContainer from './filterGroupContainer';
+import SearchBar from '../templates/SearchBar';
+
+export default class Search extends React.Component {
+  static propTypes = {
+    onChange: PropTypes.func.isRequired
+  };
+
+  handleChange = e => {
+    const value = e ? e.target.value : '';
+    this.props.onChange(value.toLowerCase());
+  };
+
+  render() {
+    return (
+      <FilterGroupContainer text="Search">
+        <SearchBar
+          clearButton
+          onChange={this.handleChange}
```

Comment:
Could use a debounce here so that handleChange isn't called as often

Reply:
I think I'm doing something wrong, because I'm getting a handful of grouchy errors along the lines of ` This synthetic event is reused for performance reasons. If you're seeing this, you're accessing the property `target` on a released/nullified synthetic event. This is set to null. If you must keep the original synthetic event around, use event.persist(). See https://fb.me/react-event-pooling for more information.` 🤔

Reply:
I just added `import debounce from 'lodash/debounce';` and 
```
debouncedOnChange = debounce(this.props.onChange, 300);

  handleChange = e => {
    const value = e ? e.target.value : '';
    this.debouncedOnChange(value);
  };
  ```
  It seemed to work. Does that work for you?